lib/src/symbol_generator.dart

```diff
@@ -47,7 +47,7 @@ class SymbolGenerator {
     return [
       'scip-dart',
       _getPackage(element),
-      _getDescriptor(element),
+      descriptor,
```

Comment:
we were unnecessary generating the descriptor twice, for every symbol (see line 43) this cleans that up